django/contrib/postgres/fields/ranges.py

```diff
@@ -224,9 +233,14 @@ def get_prep_lookup(self):
 
 models.DateField.register_lookup(RangeContainedBy)
 models.DateTimeField.register_lookup(RangeContainedBy)
+models.SmallIntegerField.register_lookup(RangeContainedBy)
 models.IntegerField.register_lookup(RangeContainedBy)
 models.BigIntegerField.register_lookup(RangeContainedBy)
 models.FloatField.register_lookup(RangeContainedBy)
+models.SmallAutoField.register_lookup(RangeContainedBy)
+models.AutoField.register_lookup(RangeContainedBy)
+models.BigAutoField.register_lookup(RangeContainedBy)
```

Comment:
Are you sure these are necessary? I think they can be removed because `Field.get_lookup` respects MRO and these fields extend the appropriate `IntegerField` since 21e559495b8255bba1e8a4429cd083246ab90457.